Food/migrations/0001_initial.py

```diff
@@ -1,15 +1,25 @@
+<<<<<<< HEAD
+<<<<<<< HEAD
+# Generated by Django 3.1.3 on 2020-11-16 08:51
+=======
+=======
 
 # Generated by Django 3.1.3 on 2020-11-11 09:05
 
 from django.db import migrations, models
 
+>>>>>>> 09ee106370006735ea3a298908be157674a8bab1
 # Generated by Django 3.1.3 on 2020-11-10 09:41
+>>>>>>> a0a3a53844208a7fcf06d0d45b96a62bbe87b56a
 
 from django.db import migrations, models
 import django.db.models.deletion
+<<<<<<< HEAD
```

Comment:
For now you can add models to gitignore to avoid merge conflicts cc @NyururuKelvin